{{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html

```diff
@@ -13,13 +13,9 @@ <h1 class="subtitle" i18n>{{cookiecutter.project_title}}</h1>
                         </p>
                     </div>
                 </div>
-                <div class="column has-text-right-tablet">
-                    <a class="university-logo image" href="https://uu.nl" target="_blank">
-                        <img src="{{"{{environment.assets}}"}}/utrecht-university.svg" alt="Utrecht University logo" i18n-alt width="200"
-                            height="50" />
-                    </a>
-                    <a class="lab-logo image" href="https://dig.hum.uu.nl" target="_blank">
-                        <img src="{{"{{environment.assets}}"}}/dhlab.svg" alt="Digital Humanities Lab logo" i18n-alt width="200" height="40" />
+                <div class="column has-text-right-tablet is-narrow">
+                    <a class="lab-logo image" href="https://cdh.uu.nl/centre-for-digital-humanities/research-software-lab/" target="_blank">
+                        <img src="{{"{{environment.assets}}"}}/uu-cdh.png" alt="Centre for Digital Humanities, Utrecht University" i18n-alt width="200" height="40" />
```

Comment:
Would an SVG version be feasible? Nicer for high resolution displays.

Reply:
Good one. There are no SVGs in the [branding](https://github.com/CentreForDigitalHumanities/Branding) repository, I'll ask if we can get one.